sdk/python/feast/cli.py

```diff
@@ -371,5 +371,58 @@ def sync_offline_to_online(feature_table: str, start_time: str, end_time: str):
     feast.pyspark.aws.jobs.sync_offline_to_online(client, table, start_time, end_time)
 
 
+@cli.command()
+@click.option(
+    "--feature-table",
+    "-t",
+    help="Feature table name to ingest data into",
+    required=True,
+)
+@click.option(
+    "--jar", "-j", help="Feature table name to ingest data into", default="",
+)
+def start_stream_to_online(feature_table: str, jar: str):
+    """
+    Start stream to online sync job.
+    """
+    import feast.pyspark.aws.jobs
+
+    client = Client()
+    table = client.get_feature_table(feature_table)
+    feast.pyspark.aws.jobs.start_stream_to_online(client, table, [jar] if jar else [])
```

Comment:
Should we be bypassing the client here? What would happen in the case where we use the job service? or will that be added later?

Reply:
The job service will have to talk to the core service to get the metadata too, so it would need a client (for example, to find the feature tables so it can kick off streaming ingestion jobs)

Reply:
Oh I meant moving all requests through the client, even starting jobs.